zanata-war/src/main/java/org/zanata/action/VersionHome.java

```diff
@@ -370,7 +370,6 @@ public String createVersion() {
     }
 
     public void copyVersion() {
-        getInstance().setStatus(EntityStatus.READONLY);
```

Comment:
That may be better. Otherwise if someone get into editor or do a push, it may clobber the in progress data